src/communication/xenvchan.cpp

```diff
@@ -14,15 +14,15 @@ namespace aos::zephyr::communication {
  * Public
  **********************************************************************************************************************/
 
-aos::Error XenVChan::Init(const aos::String& xsReadPath, const aos::String& xsWritePath)
+aos::Error Transport::Init(const aos::String& xsReadPath, const aos::String& xsWritePath)
```

Comment:
Do not rename to `Transport`

Reply:
why ? in this case it reduce write in several places `#ifdef` and so on

Reply:
Done